patch_files/patches/46_resume_cancelled_cloudify_workflows_tasks.py

```diff
@@ -0,0 +1,39 @@
+diff --git a/cloudify/workflows/tasks.py b/cloudify/workflows/tasks.py
+index cf3aedc..80b14f1 100644
+--- a/cloudify/workflows/tasks.py
++++ b/cloudify/workflows/tasks.py
+@@ -421,6 +421,9 @@ class RemoteWorkflowTask(WorkflowTask):
+         # can safely be rerun
+         if state == TASK_SENDING:
+             return
++        if self.cloudify_context['executor'] != 'host_agent' \
```

Comment:
changes in this file are specific to this patch and are not in master, unfortunate

those changes have to do with how we aren't going to patch agents, and so for agent (not mgmtworker) operations it still has to be the dispatcher side that updates the operation state